website/docs/d/eks_node_group.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "EKS"
+layout: "aws"
+page_title: "AWS: aws_eks_node_group"
+description: |-
+  Retrieve information about an EKS Node Group
+---
+
+# Data Source: aws_eks_node_group
+
+Retrieve information about an EKS Node Group.
+
+## Example Usage
+
+```hcl
+data "aws_eks_node_group" "example" {
+  cluster_name    = "example"
+  node_group_name = "example"
+}
+
+output "node_group_arn" {
+  value = "${data.aws_eks_node_group.example.node_role_arn}"
+}
+
+output "subnet_ids" {
+  value = "${data.aws_eks_node_group.example.subnet_ids}"
+}
+```
+
+## Argument Reference
+
+* `cluster_name` - (Required) The name of the cluster.
+* `node_group_name` - (Required) The name of the node group.
+
+## Attributes Reference
+
+* `id` - EKS Cluster name and EKS Node Group name separated by a colon (`:`).
+* `ami_type` - Type of Amazon Machine Image (AMI) associated with the EKS Node Group.
+* `arn` - Amazon Resource Name (ARN) of the EKS Node Group.
+* `disk_size` - Disk size in GiB for worker nodes.
+* `instance_types` - Set of instance types associated with the EKS Node Group.
+* `labels` - Key-value map of Kubernetes labels. Only labels that are applied with the EKS API are managed by this argument. Other Kubernetes labels applied to the EKS Node Group will not be managed.
```

Comment:
This is a data resource: I don't believe any of these labels are managed?

Reply:
Those are the labels assigned to the node groups, that comment is basically the `labels` description on the Node Group documentation https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/eks_node_group#labels

The data source will just return all the attributes it can gather from the specific resource